.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -860,14 +860,30 @@ tsh bench sessions --max=5000 --web user ls
 
 ## TLS Routing
 
-- [ ] Verify that teleport proxy `v2` configuration starts only a single listener.
+- [ ] Verify that teleport proxy `v2` configuration starts only a single listener for proxy service, in contrast with `v1` configuration.
+  Given configuration:
   ```
   version: v2
-  teleport:
-    proxy_service:
-      enabled: "yes"
-      public_addr: ['root.example.com']
-      web_listen_addr: 0.0.0.0:3080
+  proxy_service:
+    enabled: "yes"
+    public_addr: ['root.example.com']
+    web_listen_addr: 0.0.0.0:3080
+  ```
+  There should be total of three listeners, with only `*:3080` for proxy service:
+  ```
+  lsof -i -P | grep teleport | grep LISTEN
+    teleport  ...  TCP *:3025 (LISTEN)
```

Comment:
I don't think that 3025 should be here. If you run all services on the same box, then yes, but in a "normal prod scenario," this won't be the case. I think that this scenario should be more specific, or let me know if I'm making it over complicated.

Reply:
I don't want to make the setup too complex, this is a rather simple test at the end of the day. Any suggestions for alternative configs here?

Reply:
Maybe we can state that the proxy_service should expose only port 3080 but depends on the configuration the auth_service, ssh_service configuration the 3025 and 3022 ports can be also presents.